cmd/promtool/main.go

```diff
@@ -235,12 +235,14 @@ func main() {
 
 	tsdbDumpCmd := tsdbCmd.Command("dump", "Dump samples from a TSDB.")
 	dumpPath := tsdbDumpCmd.Arg("db path", "Database path (default is "+defaultDBPath+").").Default(defaultDBPath).String()
+	dumpSandboxDir := tsdbDumpCmd.Flag("sandbox-dir", "Sandbox directory. Cleaned up at the end.").Default("").String()
```

Comment:
I don't know if I should be more explicit here? maybe in promtool docs?

Reply:
maybe mention `Sandbox directory in case wal replay generates chunks. Cleaned up at the end`

Reply:
How about `Sandbox directory in case of transient file writes. Cleaned up at the end` to keep it generic?